pkgs/development/tools/misc/elfutils/default.nix

```diff
@@ -39,15 +39,19 @@ stdenv.mkDerivation rec {
     })
   ] ++ lib.optionals stdenv.hostPlatform.isMusl [ ./musl-error_h.patch ];
 
-  outputs = [ "bin" "dev" "out" "man" ];
+  postPatch = ''
+    for t in tests/*.sh; do
+      substituteInPlace $t --replace "/usr/bin/env bash" "${bash}/bin/bash"
+    done
```

Comment:
I think we can just do `patchShebangs tests/*.sh` and no need to add bash explicitly